zipkin-lens/src/models/AdjustedTrace.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { ServiceNameAndSpanCount } from './TraceSummary';
+
+export type AdjustedAnnotation = {
+  value: string;
+  timestamp: number;
+  endpoint: string; // If not exist, this filed will be 'unknown'.
```

Comment:
```suggestion
  endpoint: string; // Ex. 'fooo' or 'unknown' on null span.localEndpoint
```

where 'foo' is a real example